app/src/main/java/com/github/houseorganizer/houseorganizer/calendar/MonthlyAdapter.java

```diff
@@ -93,6 +102,19 @@ private void prepareMonthlyView(MonthlyDayViewHolder holder, int position) {
         }
     }
 
+    @SuppressLint("NotifyDataSetChanged")
+    public void moveMonth(int monthsDifference, TextView yearMonthText) {
```

Comment:
I think you can use `plusMonths` for both cases, it says [here](https://docs.oracle.com/javase/8/docs/api/java/time/YearMonth.html#plusMonths-long-) that it's allowed? I'm not sure if this is a better option though, so feel free to keep the code as-is.

Reply:
Oh I just assumed it wasnt possible as both methods exist, ill try to change it in the next commit then, thanks for the remark !